rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -1582,6 +1380,13 @@ def _enrich_protocol_transfers(self, context: EnricherContext) -> TransferEnrich
 
         return FAILED_ENRICHMENT_OUTPUT
 
+    def _handled_decode_exceptions(self) -> tuple[type[Exception], ...]:
+        return super()._handled_decode_exceptions() + (
+            NotERC721Conformant,
+            NotERC20Conformant,
+            Web3Exception,
+        )
+
```

Comment:
i mean, worst case you call it once in the init and store it in a variable. No need to have a function call everytime